src/frontends/paddle/src/op/pool3d.cpp

```diff
@@ -0,0 +1,202 @@
+//*****************************************************************************
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+//*****************************************************************************
+
+#include "openvino/frontend/paddle/node_context.hpp"
+#include "openvino/opsets/opset6.hpp"
+#include "openvino/opsets/opset8.hpp"
```

Comment:
why include different version of opset header file. Can only include "default_opset.hpp" here?

Reply:
Update. Most of them have used the default opset, But when I use default_opset MaxPool, strange errors will appear.

![image](https://github.com/openvinotoolkit/openvino/assets/27223114/3369f9e5-9bbd-4842-ac78-a62c3c6e813b)

Reply:
https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/op/max_pool.hpp , MaxPool has two kinds of class v1 and v8. It's not accuracy to use opset6 for MaxPool here. When you use opset6, the MaxPool should use v1. The default_opset will use the latest class v8. The v8 MaxPool has two outputs, https://docs.openvino.ai/2022.3/openvino_docs_ops_pooling_MaxPool_8.html#doxid-openvino-docs-ops-pooling-max-pool-8 . Maybe it is the problem?

Reply:
Thank you, I figure out, Now Fixed